### PR TITLE
chore(Java): add prime string

### PR DESCRIPTION
**Pull request by SamruddhiR**
**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [ ] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [ ] My submission has the time complexity of the algorithm
- [x] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] New algorithm
- [ ] Optimization in previous algorithms
- [x] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

Created a pr for issue #588

### CODE REVIEW

File: algorithms/Java/strings/PrimeString.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+		         
+		        String S=sc.next();
+		        S=S.toLowerCase();
+		        int len=S.length();
+		       
+			     for(int i=0;i<len;i++)
+		        {
+		           if((i+1)%2==0)
+		           {
+		            ch=S.charAt(i);
```
Comment (aayushjain7) [inline algorithms/Java/strings/PrimeString.java:56]: fix indentation

**Review by aayushjain7 — CHANGES_REQUESTED** [review]
Also update readme.MD file.

File: algorithms/Java/strings/PrimeString.java (hunk 1 above)
Comment (SamruddhiR) [reply algorithms/Java/strings/PrimeString.java:56]: Ok. I will do it

Comment (SamruddhiR) [reply algorithms/Java/strings/PrimeString.java]: I have made the changes. Can u please check it once.Thankyou

**Review by aayushjain7 — CHANGES_REQUESTED** [review]
update readme.MD file also

**Comment by SamruddhiR** [comment]
> Also update readme.MD file.

Ok.I will do it

**Comment by ming-tsai** [comment]
Hi @SamruddhiR,
Any udpated?

**Comment by ming-tsai** [comment]
Hi @SamruddhiR,
This pull request will be closed, any updates please reopen this pull request.

**Pull request closed without merging**